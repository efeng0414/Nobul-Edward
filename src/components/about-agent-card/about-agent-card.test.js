import React from "react";
import renderer from "react-test-renderer";
import { Card } from "antd";
import AboutAgentCard from "./index";

describe("AboutAgentcard", () => {
  const componentProps = {
    agentImage: "fakeImage",
    agentRating: 123,
    agentName: "fakeName",
    brokerage: "fakeBrokerage"
  };
  const componentJSX = <AboutAgentCard {...componentProps} />;

  describe("Component Snapshot", () => {
    const componentSnapshot = renderer.create(componentJSX).toJSON();
    it("should match existing snapshot", () => {
      expect(componentSnapshot).toMatchSnapshot();
    });

    describe("Card component", () => {
      const componentMount = global.Enzyme.mount(componentJSX);
      it("always renders a 'Card' component", () => {
        expect(componentMount.find(Card).length).toBe(1);
      });
    });
  });
});
