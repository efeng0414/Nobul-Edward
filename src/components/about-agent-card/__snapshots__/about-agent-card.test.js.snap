// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutAgentcard Component Snapshot should match existing snapshot 1`] = `
<div
  className="ant-card about-agent-card ant-card-bordered"
>
  <div
    className="ant-card-body"
    style={Object {}}
  >
    <h2>
      About this agent
    </h2>
    <div
      className="flex"
    >
      <div
        className="about-agent-card-grid-item"
      >
        <img
          alt="Agent Image"
          className="about-agent-card-profile-image"
          src="fakeImage"
        />
      </div>
      <div
        className="about-agent-card-grid-item"
      >
        <p>
          fakeName
        </p>
        <p
          className="about-agent-card-brokerage"
        >
          fakeBrokerage
        </p>
      </div>
    </div>
  </div>
</div>
`;
